### DISCUSSION TITLE
Comments

### DISCUSSION THREAD

**Discussion by tutralex** [hardinfo2/hardinfo2#100] [Q&A] [answered]
I don't know why, but recently I can't see comments in commits if they exist and I didn't start them. However, I can see them if I use incognito mode in the browser. Maybe, in addition to permission to write, you have limited me in some other way?
![Screenshot_2024-10-26-17-38-54-809_com android chrome](https://github.com/user-attachments/assets/31c08421-a75c-46d7-a6d4-8e95a7c6e028)

**Comment by tutralex** [comment]
It doesn’t matter whether I do it from my phone or computer.

**Comment by hwspeedy** [comment]
@tutralex 
No the write->read permission change is general organization change - not specific to any users. - Actually this is the normal mode for any FOSS repo.

As you can see comments in incognito mode in the browser it is probably cookies - try remove all cookies and log back in to github.

PS: I can see the code comments without even logging in - public available.

Mayby github is having problems...

**Comment by hwspeedy** [comment]
PPS: Cannot see code comments on my phone - that's not implemented.

**Comment by tutralex** [comment]
I cleared all cookies, it didn't help, comments are implemented on my phone. If the first comment is mine, I see it, if it's yours, I don't. In incognito mode, I see everything. I noticed this behavior yesterday.
![Screenshot_2024-10-26-20-41-57-743_com android chrome](https://github.com/user-attachments/assets/1fc53941-f1f6-4707-a1f3-79fec9ce2d7c)
![Screenshot_2024-10-26-20-39-39-719_com android chrome](https://github.com/user-attachments/assets/04627255-c0fc-4a5c-8c58-2639b1f7b53e)

**Comment by tutralex** [comment]
I reset cookies, then did two-factor authentication on PC, same behavior as on phone described above. Regardless of device - probably GitHub's problems.

**Comment by tutralex** [comment]
My comment is the first, then everything is visible.
![scr-1](https://github.com/user-attachments/assets/f0eaa7a2-5be8-425f-89cc-fbba013380d5)
If the first comment is yours, then it is not visible and you cannot add new ones. There is no field for entering a comment.
![scr-2](https://github.com/user-attachments/assets/befce4d9-eebf-4857-a8f6-81697dc2405a)

**Comment by tutralex** [comment, marked as answer]
**Support answer:**
There might be a temporary issue with GitHub's servers. Try refreshing the page or come back later to see if the issue has been resolved.
